T/Transducers/Compat.toml

```diff
@@ -7,5 +7,10 @@ ArgCheck = "1"
 ["0.2"]
 Setfield = "0.3-0"
 
-["0.3-0"]
+["0.3"]
 Setfield = "0.3.0-*"
+
+["0.4-0"]
+BangBang = "*"
```

Comment:
From the commit message on your repo it looks like this is not what you intended

Reply:
> Is this compat entry invalid?

It is valid, but the union of `>0.2` and `<1` is everything (`"*"`).

> But other compat entry like Setfield = "≥ 0.3, < 1" seems to be working.

As you can see that also allows any version (`"*"`).